Update steps for creating local/remote file repo (#1650)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-remote-file-repository.adoc

To create a file type repository in a remote directory, complete the following procedure:

ifdef::satellite[]
. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.
** For {EL} 8:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL8BaseOS} \
--enable={RepoRHEL8AppStream} \
--enable={RepoRHEL8ServerSatelliteUtils}
----
** For {EL} 7:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL7Server} \
--enable={RepoRHEL7ServerSatelliteUtils}
----
endif::[]
. Install the Pulp Manifest package:
** For {EL} 8:
+
[options="nowrap" subs="+quotes,attributes"]
----
# {package-install-project} python3-pulp_manifest
----
** For {EL} 7:
+
[options="nowrap" subs="+quotes,attributes"]
----
# {package-install-project} tfm-pulpcore-python3-pulp_manifest
----
. Create a directory that you want to use as the file type repository in the HTTP server's public folder:

> +

Reviewer: Remove `+`.

To create a file type repository in a remote directory, complete the following procedure:

ifdef::satellite[]

> . On your remote server, ensure that the Server and {project-client-name} repositories are enabled.

Reviewer 2: Should we change `Server` here to one of the EL macros? Server can be confusing especially given RHEL 8 calls it Base OS and AppStream  @ekohl

Reviewer 3: Perhaps not even specify the names at all. Just say:
```suggestion
. On your machine, ensure that the right repositories are enabled.
```

This is because utils could in theory also run from a desktop (at least with EL8).

** For {EL} 8:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL8BaseOS} \
--enable={RepoRHEL8AppStream} \
--enable={RepoRHEL8ServerSatelliteUtils}
----
** For {EL} 7:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL7Server} \
--enable={RepoRHEL7ServerSatelliteUtils}
----
endif::[]
. Install the Pulp Manifest package:
** For {EL} 8:
+
[options="nowrap" subs="+quotes,attributes"]
----

> # {package-install-project} python3-pulp_manifest

Reviewer: ```suggestion
# {package-install} python3-pulp_manifest
```
The same attribute as [above](#diff-c4152de0d96ec7f6a12947f504675387578d2cfc56fa48871b82a6fbe6c1bb4fL40-R39) perhaps?

----
** For {EL} 7:
+
[options="nowrap" subs="+quotes,attributes"]
----

> # {package-install-project} tfm-pulpcore-python3-pulp_manifest

Reviewer 4: See https://github.com/theforeman/foreman-documentation/pull/1671#discussion_r991870096

. On your remote server, ensure that the Server and {project-client-name} repositories are enabled.

> ifdef::satellite[]

Reviewer 3: I think these should be swapped.

** For RHEL 8:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL8BaseOS} \
--enable={RepoRHEL8AppStream} \
--enable={RepoRHEL8ServerSatelliteUtils}
----
** For RHEL 7:
+
[options="nowrap" subs="+quotes,attributes"]
----
# subscription-manager repos --enable={RepoRHEL7Server} \
--enable={RepoRHEL7ServerSatelliteUtils}
----
endif::[]
. Install the Pulp Manifest package:

> ** For RHEL 8:

Reviewer 3: ```suggestion
** For {EL} 8:
```

+
[options="nowrap" subs="+quotes,attributes"]
----
# {package-install-project} python3-pulp_manifest
----

> ** For RHEL 7:

Reviewer 3: ```suggestion
** For {EL} 7:
```
